workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/BodyAndOverlaysContainer.kt

```diff
@@ -84,11 +84,11 @@ internal class BodyAndOverlaysContainer @JvmOverloads constructor(
   }
 
   override fun dispatchTouchEvent(event: MotionEvent): Boolean {
-    return !dialogs.allowEvents || super.dispatchTouchEvent(event)
+    return !dialogs.allowContentEvents || super.dispatchTouchEvent(event)
```

Comment:
Unrelated method rename to try to make it clearer that this is strictly about event handling in the activity window, not the dialogs.

Reply:
Why `allowContentEvents` and not `allowBodyEvents`? I think Body would help me understand what you mean.

Reply:
I wrestled with that. I went with `content` because it matches `ScreenOverlay.content`, which in turn comes from the `Wrapper` interface.  But really they're both still ambiguous, aren't they -- dialog content? dialog body?

How about `allowBaseEvents`? That would match the `updateBase` param to `LayeredDialogSessions.update`.

Reply:
See insane stream of consciousness below, `allowBodyEvents` and `updateBody()` FTW.